src/components/TaskCard/index.jsx

```diff
@@ -1,30 +1,53 @@
 import PropTypes from "prop-types";
 import { useDispatch } from "react-redux";
-import Button from "@components/Button";
-import { deleteTask } from "@store/actions";
-import { ICON_DELETE } from "@utils/constants/icons";
-import { ALT_TEXT_DELETE_ICON } from "@utils/constants/texts";
+import { deleteTask, setTaskDone } from "@store/actions";
+import { ICON_DELETE, ICON_DONE } from "@utils/constants/icons";
+import { ALT_TEXT_DELETE_ICON, ALT_TEXT_DONE_ICON } from "@utils/constants/texts";
 import { formatDate } from "@utils/helpers/formatDate";
+import { compareDates } from "@utils/helpers/compareDates"
+import Button from "@components/Button";
 import "./index.scss"
 
 const TaskCard = ({ task }) => {
-  const { id, title, createdDate } = task;
+  const { id, title, createdDate, completedDate, isDone } = task;
   const dispatch = useDispatch();
 
   const handleDelete = () => {
     dispatch(deleteTask(id));
   };
 
+  function handleDone(){
+    dispatch(setTaskDone(id));
+  };
+
+  const getDaysToCompleteTask = (startDate, endDate) => {
+    const diffInDays = compareDates(startDate, endDate);
+    const daysStr = diffInDays > 1 ? `${diffInDays} days` : `${diffInDays} day`;
+
+    return daysStr;
+  };
+
   return (
     <div className="task-card">
-      <p className="task-card__title">{title}</p>
+      <p className={`${isDone ? "task-card--done__title" : "task-card__title"}`}>
```

Comment:
`isDone` is quite difficult to understand. Please name it to `isTaskDone` || `isTaskCompleted`
Also, use classnames package here
{
"task-card--done__title" : isDone,
"task-card__title" : !isDone,
}